### ISSUE TITLE
Packet prioritization is not supported.

### ISSUE THREAD

**Issue by Philipel-WebRTC** [w3c/webrtc-rtptransport#38]
In UC1 there is no mechanism to set the priority of a packet.

In UC2 there app decides the send order, but there is no way to set the RTP sequence number.

**Comment by tonyherre** [comment]
Do you need modifiable sequence numbers for prioritization? Deciding send order should be sufficient?

**Comment by Philipel-WebRTC** [comment]
You are right that for UC2 we probably wouldn't need to set the RTP sequence number from the app to do packet prioritization. I updated my initial comment.
